<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleSetting</class>
 <widget class="QDialog" name="ScaleSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting Scale</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>10</y>
     <width>183</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X-scale          </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xScaleLine">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y-scale          </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yScaleLine">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Z-scale          </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zScaleLine">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Deform Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deformScaleLine">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
